rules/_deprecated/persistence_cron_jobs_creation_and_runtime.toml

```diff
@@ -1,7 +1,8 @@
 [metadata]
 creation_date = "2021/01/15"
-maturity = "production"
-updated_date = "2021/03/03"
+deprecation_date = "2021/04/15"
+maturity = "deprecated"
```

Comment:
this is a big persistence mechanism so it's sad to see it go, but we don't have much choice if the noise level is high and unpredictable. And we can't really solve that much by narrowing the scope

I wonder if an ML job would be a better fit for this TTP

Reply:
Yeah, this one can be very random per environment, a rare ML job would be the best option.